src/components/Profile/components/SchedulePanel/index.jsx

```diff
@@ -47,33 +43,23 @@ class GordonSchedulePanel extends Component {
     super(props);
     this.state = {
       myProf: false, //myProf is boolean value that determines whether this is myprofile or not. this.props.profile actually contains profile data.
-      isSchedulePrivate: 0,
       isExpanded: false,
-      myScheduleOpen: false,
       disabled: true,
-      selectedEvent: null,
+      selectedEvent: null, //see if we actually need this later
```

Comment:
```suggestion
      selectedEvent: null, //see if we actually need this later
```

did you figure out if you needed it? if so, remove the comment, otherwise remove `selectedEvent`